federation/ldap/src/main/java/org/keycloak/storage/ldap/mappers/HardcodedLDAPGroupStorageMapperFactory.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2016 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.keycloak.storage.ldap.mappers;
+
+import org.keycloak.component.ComponentModel;
+import org.keycloak.component.ComponentValidationException;
+import org.keycloak.models.GroupModel;
+import org.keycloak.models.KeycloakSession;
+import org.keycloak.models.RealmModel;
+import org.keycloak.models.utils.KeycloakModelUtils;
+import org.keycloak.provider.ProviderConfigProperty;
+import org.keycloak.storage.ldap.LDAPStorageProvider;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * @author <a href="mailto:jean-loup.maillet@yesitis.fr">Jean-Loup Maillet</a>
+ */
+public class HardcodedLDAPGroupStorageMapperFactory extends AbstractLDAPStorageMapperFactory {
+
+    public static final String PROVIDER_ID = "hardcoded-ldap-group-mapper";
+    protected static final List<ProviderConfigProperty> configProperties = new ArrayList<ProviderConfigProperty>();
+
+    static {
+        ProviderConfigProperty groupAttr = createConfigProperty(HardcodedLDAPGroupStorageMapper.GROUP, "Group",
+                "Group to add the user in.  Click 'Select Group' button to browse groups, or just type it in the textbox.",
```

Comment:
I think this tooltip, which will be shown in the admin console, is not correct. The type of the configuration property is String, so there won'be anything like "Select Group" combobox if I am correct? User will need to type the group manually including group path. Could you change the tooltip to something like: 
``
Group to add the user in.  Fill the full path of the group including path. For example '/root-group/child-group' . 
``
WDYT?

If you want, you can contribute "Group Selector" angular component to our admin console. Something like "Role Selector" is currently doing. But that would be separate PR (and separate JIRA) if you want to do this :)

Reply:
Yes, you are perfectly right.
I wanted to do the group selection component but postponed it... Did not think of correcting the tooltip.